shopinvader_product_description/models/product_template.py

```diff
@@ -0,0 +1,12 @@
+# Copyright 2023 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ProductTemplate(models.Model):
+
+    _inherit = "product.template"
+
+    short_description = fields.Html()
+    description = fields.Html()
```

Comment:
I have made https://github.com/OCA/product-attribute/tree/16.0/product_sale_description for that